exercises/simple-linked-list/src/test/scala/SimpleLinkedListTest.scala

```diff
@@ -0,0 +1,56 @@
+import org.scalatest.{FlatSpec, Matchers}
+
+class SimpleLinkedListTest extends FlatSpec with Matchers {
```

Comment:
I'll take a look at the Haskell tests.
I started with the C# tests as my template.

Another opportunity to add a canonical-data.json :)